Makefile.am

```diff
@@ -9,6 +9,7 @@ include_HEADERS = libipl/libipl.H
 
 if BUILD_PHAL_API
 lib_LTLIBRARIES += libphal.la
+include_HEADERS += libphal/exception.H
```

Comment:
The header files which are installed should be unique or they should be installed in a sub-directory.   

I would rename `exception.H` to `phal_exception.H` to ensure uniqueness.